test/commons/aria/is-aria-role-allowed-on-element.js

```diff
@@ -223,12 +223,4 @@ describe('aria.isAriaRoleAllowedOnElement', function() {
     var actual = axe.commons.aria.isAriaRoleAllowedOnElement(node, 'link');
     assert.isTrue(actual);
   });
-
-  it('returns false if elements implicit role does not match the role', function() {
```

Comment:
I don't think this test adds any value. It doesn't return false for just this reason, it returns false because none of the things we test for that could make it true are. So I decided to remove it rather than put in some other element.